cli/kube_operator/kube_deployment.go

```diff
@@ -0,0 +1,165 @@
+package kube_operator
```

Comment:
Please change the name of this package to kube_deployment to be consistent with the rest of the plugins.

Reply:
done